src/screens/TopicPageScreen/index.js

```diff
@@ -44,7 +44,7 @@ const TopicPageScreen = (props) => {
   const [isFollow, setIsFollow] = React.useState(params.isFollowing);
   const [followType, setFollowType] = React.useState('');
   const [topicDetail, setTopicDetail] = React.useState({});
-  const [memberCount, setMemberCount] = React.useState(params.memberCount);
+  const [memberCount, setMemberCount] = React.useState(params.memberCount || 0);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change is good as it handles the case where `params.memberCount` might be null or undefined. This will prevent potential crashes due to trying to set an undefined value to state.
```diff
- const [memberCount, setMemberCount] = React.useState(params.memberCount);
+ const [memberCount, setMemberCount] = React.useState(params.memberCount || 0);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->